src/main/java/liquibase/ext/neo4j/change/InsertNodeChange.java

```diff
@@ -0,0 +1,72 @@
+package liquibase.ext.neo4j.change;
+
+import liquibase.change.ChangeMetaData;
+import liquibase.change.ColumnConfig;
+import liquibase.change.DatabaseChange;
+import liquibase.change.core.InsertDataChange;
+import liquibase.database.Database;
+import liquibase.exception.ValidationErrors;
+import liquibase.ext.neo4j.database.Neo4jDatabase;
+import liquibase.statement.SqlStatement;
+import liquibase.statement.core.RawParameterizedSqlStatement;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static liquibase.ext.neo4j.change.ColumnMapper.mapValue;
+
+@DatabaseChange(name = "insert", priority = ChangeMetaData.PRIORITY_DATABASE,
+        description = "Inserts a node to the graph")
+public class InsertNodeChange extends InsertDataChange {
+
+    private String labelName = "";
+
+    @Override
+    public ValidationErrors validate(Database database) {
+        ValidationErrors errors = new ValidationErrors(this);
+        if (!(database instanceof Neo4jDatabase)) {
```

Comment:
Instead of validate() failing for non-neo4j databases, you should override the supports() method to say it only supports neo4j. 

With this setup, what happens is that if a user has the neo4j extension in their system and tries to do an `<insert>` against a non-neo4j database it will fail with the validation error instead of using the non-neo4j code.

Reply:
Fixed in 99cafd81900bb80d2c33ed9a94cb237b0b7bcddd